network/load-balancer/how-to/create-load-balancer.mdx

```diff
@@ -21,37 +21,45 @@ dates:
     <Lightbox src="scaleway-lb-creation-page.webp" />
 2. Click **Create a Load Balancer**. The creation wizard displays.
-3. Complete the following steps in the wizard: 
-    - Enter the Load Balancer **name** and **description**. Optionally, you can assign **tags** to help organize your Load Balancers.
-    - Choose the **Availability Zone** in which your Load Balancer will be deployed geographically. Currently we provide the following Availability Zones:
+
+3. Enter a **name** and **description** for your Load Balancer. Optionally, you can assign **tags** to help organize your Load Balancers.
+    <Lightbox src="scaleway-load-balancer-name.webp" alt="" />
+
+4. Choose the **Availability Zone** in which your Load Balancer will be deployed geographically. Currently we provide the following Availability Zones:
       - `PAR1`: Paris 1, France.
       - `PAR2`: Paris 2, France (innovative and sustainable availability zone).
       - `AMS1`: Amsterdam, The Netherlands.
       - `WAW`: Warsaw, Poland.
-      <Lightbox src="scaleway-lb-az.webp" />
-    - Select a Load Balancer **type**.
-    - Select an **IP address** for your Load Balancer. You can either allocate a new IP, or select one of your existing IPs.
-    - Configure the frontend and backend.
-      - **Configure frontend**. Enter a name an a port for your [frontend](/network/load-balancer/concepts/#frontends). The frontend will listen on this port, and forward received requests to the backend(s). Note that you cannot add an SSL certificate at this stage, but you can [add one after the creation of the Load Balancer](/network/load-balancer/how-to/setup-ssl-offloading/).
-      - **Configure backend**. Enter the following information for your [backend](/network/load-balancer/concepts/#backends):
-        - Backend name
-        - [Protocol](/network/load-balancer/concepts/#protocol): select TCP or HTTP depending on your required usage. The protocol is used for data transmission between the frontend/s and backend/s.
-        - Port: enter the port your backend will listen on.
-        - Proxy: select a [proxy protocol](/network/load-balancer/concepts/#proxy-protocol) option from the list. Depending on the type of health check you choose, you are prompted for additional information, for example HTTP healthchecks require a method (GET, PUT or POST, a URI and a response code).
-        - Server IPs: enter the IP addresses of each of your backend servers.
-    - Configure **advanced settings** if required:
-      - **S3 failover**: activate the feature and enter a Scaleway Object Storage [Bucket Website](/storage/object/concepts/#bucket-website) URL. If none of your backend servers are available, users will be redirected to this page. 
 
+5. Select a Load Balancer **type**, according to your bandwidth needs.
```

Comment:
FYI we will soon provide new metrics to help choose between LB sizes, maybe an update will be required at that time.